pallets/motion/Cargo.toml

```diff
@@ -0,0 +1,49 @@
+[package]
```

Comment:
Can remove this pallet before the migration

Reply:
Removed